app/scripts/models/apps.js

```diff
@@ -0,0 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * A collection of OAuth apps
```

Comment:
Same w/ this filename, apps is a bit too generic esp if this is only used for oauth. Can you add the OAuth prefix?